### ISSUE TITLE
Unable to build container image asset-transfer-basic/chaincode-java

### ISSUE THREAD

**Issue by tbondarchuk** [hyperledger/fabric-samples#940]
I was following https://github.com/hyperledger/fabric-samples/blob/1e8fd5cd5647c9cb6e7e5ccc4c5427153f4aebbb/test-network-k8s/README.md#L48

but docker build fails with:

```
#14 17.59 An exception occurred applying plugin request [id: 'application']
#14 17.59 > Failed to apply plugin class 'com.github.jengelman.gradle.plugins.shadow.ShadowJavaPlugin'.
#14 17.59    > You can't map a property that does not exist: propertyName=classifier
```

**Comment by satota2** [comment]
Hi @tbondarchuk
I faced the same case. The cause is because the gradle version in the gradle:jdk11-alpine image is updated from 7 to 8, which seems to cause the build to fail.
I fixed the version to 7 by changing gradle:jdk11-alpine in the Dockerfile to 7-jdk11-alpine and it worked fine.

I tried to make a patch that solves this problem and submit it.
https://github.com/hyperledger/fabric-samples/pull/961
